README.md

```diff
@@ -1,20 +1,21 @@
 # Pokemon hunt
-> Website for the annual pokemon hunt, held at Ilex Forest, nearby the wonderful Goldenrod City
+> Website for the annual pokemon hunt, held at Ilex Forest, nearby the wonderful Goldenrod City.
 
 ## Built With:
-- Html
-- Css
+- HTML
+- CSS
+- SASS
 - JavaScript
 
 ## Live Demo
-[Live Demo]()
+[Live Demo](https://gopxfs.github.io/Pokemon-hunt/)
```

Comment:
Your live link is functional! Well done using GitHub Pages to deploy your website 💣